input/infectionRates.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
This PR builds upon the changes in PR#48 - It would be easier review if _EITHER_ it did not include those changes, _OR_ the PR targetted that branch instead of the develop branch (and then I believe that if that branch is merged into develop and deleted, GitHub will automatically switch this PR to target develop).